<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="group_paramSet">
       <property name="title">
        <string>源文件编码格式</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>源文件编码格式：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_fileCodeFormat">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>新建行</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>FileFormat</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>ANSI</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>GBK</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>Unicode</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>UTF-8</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbl_fileCodeFormat">
            <property name="text">
             <string>文件编码格式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_fileCodeFormat">
            <property name="placeholderText">
             <string>ANSI,GBK,Unicode,UTF-8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_addFileFormat">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_deleteFileFormat">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_fileSuffix">
       <property name="title">
        <string>源文件编码格式</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lbl_fileSuffix">
            <property name="text">
             <string>源文件编码格式：</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_fileSuffix">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>新建行</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>4</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>FileSuffix</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>.h</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>.cpp</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>.cc</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>.c</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lbl_fileSuffix_2">
            <property name="text">
             <string>文件编码格式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_fileSuffix">
            <property name="placeholderText">
             <string>.h,.cpp,.cc,.c</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_addFileSuffix">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_deleteFileSuffix">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbl_filePath">
       <property name="text">
        <string>待转换文件夹路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_filePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_selectFilePath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_destFileCodeFormat">
       <property name="text">
        <string>目的文件格式：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_destCodeFormat">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>UTF-8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exec">
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
